music21/note.py

```diff
@@ -1630,10 +1643,16 @@ class Unpitched(NotRest):
     AttributeError: 'Unpitched' object has no attribute 'pitch'
     '''
 
-    def __init__(self):
-        super().__init__()
-        self.displayStep = 'C'
-        self.displayOctave = 4
+    def __init__(self, displayName=None, **keywords):
+        super().__init__(**keywords)
+
+        self.displayStep: str = 'C'
+        self.displayOctave: int = 4
```

Comment:
If possible, let's change the default to 'B', 4.  Any maybe put in some sort of "displayStepIsInferred" if it's not set, so that it can be affected by Instrument.

Reply:
display-step and display-octave are required, so I don't want to build extra layers on top we might not need just yet.

Reply:
Oh I see you mean starting from the m21 side.

Reply:
Still inclined to defer until we have a way of changing display pitch based on instrument

Reply:
(But I did change C4 to B4.)